decko-rails/lib/decko/rails.rb

```diff
@@ -1,11 +1,33 @@
 DECKO_RAILS_GEM_ROOT = File.expand_path('../../..', __FILE__)
 
+require 'rails/all'
+require 'decko/engine'
+
 module Decko
-  module Rails
+  module Rails # not sure we need this
     class << self
       def gem_root
         DECKO_RAILS_GEM_ROOT
       end
     end
   end
-end
\ No newline at end of file
+
+  if defined? ::Rails::Railtie
```

Comment:
Yeah, I thought I realized my question made no sense and withdrew it.

I think there may be cases where you run "bundle exec blah" on a rails app and don't load the application, in which case this kind of check may be necessary.

perhaps the file itself should be at decko/rails/railtie (at some point?)